README.md

```diff
@@ -118,27 +120,27 @@ Level of details in the logs, one of: `FATAL`, `ERROR`, `WARN`, `INFO`, `DEBUG`.
 
 ### `share-everyone`
 
-Automatically shares API collections created by the CI/CD task with everyone in your organization on the 42Crunch Platform. Accepted values are: `OFF`, `READ_ONLY`, `READ_WRITE`. Default is `OFF`. Note that the identity the action unders run (the owner of the API token) must have the `Share with Everyone` permission, otherwise the task will fail with a 403 error.
+Automatically shares API collections created by the CI/CD task with everyone in your organization on the 42Crunch platform. Accepted values are: `OFF`, `READ_ONLY`, `READ_WRITE`. Default is `OFF`. Note that the identity the action runs under (the owner of the API token) must have the `Share with Everyone` permission, otherwise the task will fail with a 403 error.
```

Comment:
"42Crunch platform" --> "42Crunch Platform"

"Platform" is integral part of our product's name and should be uppercase in both "42Crunch API Security Platform" and "42Crunch Platform". Only when referred simply as "the platform" should the lowercase be used.